File: 2-semestre/uri-online-judge/javascript/1043.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

let linha = lines.shift().split(" ");

let a = parseFloat(linha[0]);
let b = parseFloat(linha[1]);
let c = parseFloat(linha[2]);

if ((a < b + c) && (b < a + c) && (c < a + b)) {
    let perimetro = a + b + c;
    
    console.log("Perimetro = " + perimetro.toFixed(1));
} else {
    let trapezio = ((a + b) * c) / 2;
    
    console.log("Area = " + trapezio.toFixed(1));
}

File: 2-semestre/uri-online-judge/c/1043.c
#include <stdio.h>

int main() {
    double a, b, c, perimetro, trapezio;
    
    scanf("%lf", &a);
    scanf("%lf", &b);
    scanf("%lf", &c);
    
    if ((a < b + c) && (b < a + c) && (c < a + b)) {
        perimetro = a + b + c;
        
        printf("Perimetro = %.1lf\n", perimetro);
    } else {
        trapezio = ((a + b) * c) / 2;
        
        printf("Area = %.1lf\n", trapezio);
    }
    
    return 0;
}
